blockchain/genesis.go

```diff
@@ -318,6 +320,10 @@ func (g *Genesis) ToBlock(baseStateRoot common.Hash, db database.DBManager) *typ
 				continue
 			}
 		}
+		// for test
+		if len(account.CodeHash) != 0 {
```

Comment:
Just in case, please sanitize `if len(account.Code) != 0 && len(account.CodeHash) != 0 { logger.Crit("code and codehash cannot be simultaneously specified", "addr", addr.String()) }`